api/views/workflow.py

```diff
@@ -10,8 +10,12 @@
 from share.models import ChangeSet, Change, RawData, ShareUser, NormalizedData
 from share.models.validators import JSONLDValidator
 from share.tasks import MakeJsonPatches
+from share.harvest.harvester import Harvester
+from share.normalize.v1_push import V1Normalizer
+from django.db import transaction
```

Comment:
Imports should be ordered by stdlib, 3rd part, application and separated by a single blank line.

```python
import os  # stdlib

from django import thing  # 3rd party

from share.whatever import bar  # application
```